pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.1.0-beta.3"
+version = "1.1.0"
```

Comment:
Be sure that the `version` in `next` (currently `1.2.0-alpha.1`) doesn't get overwritten by your diffs.

Maybe try merging latest `next` into your branch and see if it resolves all of the extraneous diffs showing in this PR?